### ISSUE TITLE
Biomes bug

### ISSUE THREAD

**Issue by doesprintfwork** [Matocolotoe/Skript-1.8#6]
Some biomes have changed the name and I guess newer version skript also changed them. I can't check if players are in those biomes in 1.8.8.

**Comment by Matocolotoe** [comment]
Which of these biomes are causing a problem ?

**Comment by doesprintfwork** [comment]
Those like extreme hills+, extreme hills+ m, also beaches iirc. There are a bunch of them. I think all of the biomes that changed its name at 1.13 are having problem.

**Comment by Matocolotoe** [comment]
Alright, will take a look at it, thanks ^^
If it's not fixed once the new repository is done, please open another issue there.
